src/poly.c

```diff
@@ -225,6 +225,7 @@ static void _span_fill(twin_pixmap_t *pixmap,
     }
 }
 
+#define MIN(A, B) ((A) < (B) ? (A) : (B))
```

Comment:
Define `min` macro in `include/twin_private.h`. The possible definition would be as following: (for clang and gcc)
```c
#define min(x,y) ({ \
	typeof(x) _x = (x);	\
	typeof(y) _y = (y);	\
	(void) (&_x == &_y);	\
	_x < _y ? _x : _y; })
```

Reference:
* [arch/powerpc/boot/types.h](https://github.com/torvalds/linux/blob/master/arch/powerpc/boot/types.h)
* [Notes on Linux kernel's minmax](https://hackmd.io/@sysprog/linux-macro-minmax)